src/views/Header.js

```diff
@@ -227,8 +272,8 @@ class Header extends React.Component<DefaultProps, HeaderProps, *> {
     return (
       <Animated.View {...rest} style={[styles.container, style]}>
         <View style={styles.appBar}>
-          {leftComponents}
           {titleComponents}
+          {leftComponents}
```

Comment:
Because title component has left/right 40 absolute position and covers the back button title. Changed the order to make it appear.